### PR TITLE
[PAUSED] modularize cli

### PR DESCRIPTION
**Pull request by bergman**

### CODE REVIEW

File: styx-cli/src/main/java/com/spotify/styx/cli/BackfillCli.java [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+      if (alias != null && !alias.isEmpty()) {
+        subparser.aliases(alias);
+      }
+
+      return subparser;
+    }
+  }
+
+  static void runCommand(Main main, BackfillCommand backfillCommand)
```
Comment (danielnorberg) [inline styx-cli/src/main/java/com/spotify/styx/cli/BackfillCli.java]: Can we eliminate the circular dependency between `Main` and `{Backfill, Resource}Cli`?

Comment (bergman) [inline styx-cli/src/main/java/com/spotify/styx/cli/BackfillCli.java]: I was bothered by that too, maybe i can take another look at it.

**Comment by honnix** [comment]
Do we still want to have this PR? CLI has become very thin after splitting client.

**Comment by bergman** [comment]
I'll try to do another round on this, not sure if it still makes sense.

**Comment by honnix** [comment]
This PR is now having quite bad conflicts. Shall we close and try again if you plan to?

**Pull request closed without merging**